src/scss/_grid.scss

```diff
@@ -8,8 +8,98 @@
 	}
 
 	.o-layout {
-		@include _oLayoutGridBase;
 		@include oTypographySize(1);
+		position: relative;
+		display: grid;
+		box-sizing: border-box;
 		min-height: 100vh;
+		min-width: 100%;
+		grid-template-columns: 100%;
+		grid-template-rows: auto 1fr auto;
+		grid-gap: $_o-layout-gutter;
+		grid-template-areas:
+			"header"
+			"main"
+			"footer";
+	}
+}
+
+
+/// Grid for a landing page layout (e.g. homepage).
+/// @access private
+@mixin _oLayoutLandingGrid() {
+	.o-layout.o-layout--landing {
```

Comment:
do we need this specificity?

Reply:
We can probably rely on source order but there's no reason not to be explicit (the specificity is still low).